apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -902,6 +907,18 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
     }
 
 
+def get_default_partial_credit_type(
+    grading_method: GradingMethodType,
+) -> PartialCreditType:
+    # For backward compatibility, we need to override the default partial credit type
+    # when grading_method = ORDERED
+    return (
+        PartialCreditType.NONE
+        if grading_method is GradingMethodType.ORDERED
+        else PartialCreditType.LCS
+    )
+
+
 def test(element_html: str, data: pl.ElementTestData) -> None:
```

Comment:
Currently these automated tests are only testing a couple of hard coded cases. Did you try submitting a bunch of different answers to a few different types of 'ordered' questions just to make sure everything is working right?

I know this is not ideal, as you can see from the TODOs here, I would prefer to have some randomized fuzz testing here so we don't have to do this manually.

Reply:
excellent thanks!